### ISSUE TITLE
Integrate to VIM?

### ISSUE THREAD

**Issue by leslielg** [cyrus-and/gdb-dashboard#235]
Any expert can start a project(or already have one?) to integrate this to VIM? So we can have multi window debugger from VIM, it will be consistent, customize-able to use.

**Comment by gavinhoward** [comment]
Integration with `vim` is possible without changing any code in `gdb-dashboard`. You just need some other ubiquitous tools, such as `tmux`, `bash`, and `tmuxp`.

If you would like to learn more, you can read my blog post at https://gavinhoward.com/2020/12/my-development-environment-and-how-i-got-there/ .

**Comment by leslielg** [comment]
I already have made one by tmux and used for long time:
![image](https://user-images.githubusercontent.com/5469646/111422682-02c87580-872a-11eb-92f2-26acc65a86b3.png)

But to improve the experience, having an official, long term mantained VIM plugin version looks better option.

**Comment by gavinhoward** [comment]
Again, speaking for myself, not the author.

I could see a well-supported `vim` plugin being useful, but I am not sure that it would be best done in this repo. It might be best to do it in a separate repo, specifically for the plugin. Whether cyrus-and wants to do it himself or would prefer to let the `gdb-dashboard` community do it, I don't know.

Just my 2 cents.

**Comment by cyrus-and** [comment]
I agree with @gavinhoward (thanks for your post!), this repo is no place for such plugins. As for me implementing such thing, well, nope. But if someone decides to do that, he/she is welcome and can ping me or comment here, I'll do my best to try to help.

**Comment by gavinhoward** [comment]
@cyrus-and thank you. @leslielg to prevent bothering cyrus-and, you could email me at the address in my profile with ideas about what you want; since I already have a start in my `vim` config, I might be able to make such a plugin on a day that I get bored.

**Comment by cyrus-and** [comment]
Feel free to post here updates if you think it's something that others might benefit from.

**Comment by seanybaggins** [comment]
@leslielg Any chance you can share your configs for your setup?

**Comment by leslielg** [comment]
> @leslielg Any chance you can share your configs for your setup?

here: https://github.com/kthready/gdb_tmux

**Comment by chankim** [comment]
@leslielg Wow, it looks very good. Can you scroll up and down in each window? I'm curious if I can scroll up/down inside the source window and register window.
